tools/jit/gen_jit_dispatch.py

```diff
@@ -195,9 +220,18 @@ def emit_decl(decl):
         all_arguments_are_inputs = tuple(True for _ in arguments)
         only_tensors_are_inputs = tuple(is_tensor_arg(arg) for arg in arguments)
 
+        sized_intlist_args = [i for i, a in enumerate(arguments) if is_sized_intlist_arg(a)]
+        inds_to_replace_powerset = (set(c)
```

Comment:
Is there any reason to treat them as a set? `combinations` are without replacement, so it should be ok to use tuples.

Reply:
Nah, it was an impulse since later I test `if (value in collection)`. This ofc. doesn't matter for collections of length ~4.